src/main/resources/anniv6Resources/localization/eng/ManaSurge/Keywordstrings.json

```diff
@@ -0,0 +1,130 @@
+[
+  {
+    "ID": "sharp",
```

Comment:
Wait, do you even need to add the tooltips manually if you already defined these keywords to put onto cards?

Reply:
i did not but the way i actually knew how to do it was not available (basemod.getkeywordproper) so I just hardcoded everything until I figured it out and forgot lmao, hopefully all fixed.